src/pal/src/CMakeLists.txt

```diff
@@ -1,13 +1,18 @@
 cmake_minimum_required(VERSION 2.8.12.2)
 
+if(CMAKE_SYSTEM_NAME STREQUAL Darwin)
+  # On OSX, we use the libunwind that's part of the OS
+  set(CLR_USE_SYSTEM_LIBUNWIND 1)
+endif(CMAKE_SYSTEM_NAME STREQUAL Darwin)
+
 include_directories(SYSTEM /usr/local/include)
-include_directories(libunwind/include)
 
 add_compile_options(-fPIC)
 
-if(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
+if(NOT CLR_USE_SYSTEM_LIBUNWIND)
```

Comment:
A nit - could you please rename the symbol to CLR_CMAKE_USE_SYSTEM_LIBUNWIND? We have other symbols that we pass to cmake from our build and they are all named with CLR_CMAKE prefix, so it would be good to keep the convention.

Reply:
Sorry, I didn't realize the convention. Fixed now.